generated-sources.json

```diff
@@ -142,11 +142,18 @@
         "sha256": "41b86e89eb734e0c5ff7f2f41129a1fbbc46eadccab4c39b33c95551d99f287c",
         "dest": "flatpak-node/cache/node-gyp/27.1.3"
     },
+    {
+        "type": "archive",
+        "url": "https://www.electronjs.org/headers/v27.1.3/node-v27.1.3-headers.tar.gz",
+        "strip-components": 1,
+        "sha256": "8576e6ddd9ac2cc40eb96dcb27352bdafd29ea4199cc8f83df0d2e608122c848",
+        "dest": "flatpak-node/cache/node-gyp/27.1.3"
+    },
     {
         "type": "file",
-        "url": "https://github.com/desktop/dugite-native/releases/download/v2.39.3/dugite-native-v2.39.3-887e871-ubuntu.tar.gz",
-        "sha256": "7a6c87e0010896c6580a09a973a3ba895ebbac4f980fbf57502615954e3cd24f",
-        "dest-filename": "dugite-native-v2.39.3-887e871-ubuntu.tar.gz",
+        "url": "https://github.com/desktop/dugite-native/releases/download/v2.39.3-1/dugite-native-v2.39.3-91ebaa8-ubuntu-x64.tar.gz",
+        "sha256": "78375b97c802caa33c4ab585e3cf113001f0f53d0ab623ef0086e7c5b819189d",
+        "dest-filename": "dugite-native-v2.39.3-91ebaa8-ubuntu-x64.tar.gz",
```

Comment:
It seems dugite's prebuilt binary url is platform dependent, thus on aarch64 we cannot find it.

Reply:
https://github.com/desktop/dugite/blob/main/docs/environment-variables.md#execution

We can try to patch out the `download-git.js` script in postinstall, then we should try to use local git.